src/test/org/pantsbuild/jarjar/integration/FileTree.java

```diff
@@ -0,0 +1,70 @@
+package org.pantsbuild.jarjar.integration;
+
+import java.io.File;
+import java.util.Iterator;
+import java.util.Stack;
+
+/**
+ * Iterator over a file tree.
+ *
+ * Roughly correlates to python's os.walk.
+ */
+public class FileTree implements Iterable<String> {
```

Comment:
It's looking like it actually is about as complex to adapt guava's FileTreeTraverser to my needs as it is to just use the FileTree I have here, since I need an iterator which skips directories and returns paths relative to the root. I wrote an iterator which performs this behavior (since that's nicer than doing the folder skipping and relativizing in the caller), and stopped when I had already exceeded how much code my original FileTree had.